webui/react/src/pages/ExperimentList.tsx

```diff
@@ -712,21 +535,24 @@ const ExperimentList: React.FC = () => {
           { disabled: !hasCancelable, label: Action.Cancel, value: Action.Cancel },
           { disabled: !hasKillable, label: Action.Kill, value: Action.Kill },
         ]}
-        selectedRowCount={selectedRowKeys.length}
+        selectedRowCount={(settings.row || []).length}
         onAction={handleBatchAction}
         onClear={clearSelected}
       />
       <ResponsiveTable<ExperimentItem>
         columns={columns}
         dataSource={experiments}
         loading={isLoading}
-        pagination={getFullPaginationConfig(pagination, total)}
+        pagination={getFullPaginationConfig({
+          limit: settings.tableLimit,
+          offset: settings.tableOffset,
+        }, total)}
         rowClassName={defaultRowClassName({ clickable: false })}
         rowKey="id"
         rowSelection={{
           onChange: handleTableRowSelect,
           preserveSelectedRowKeys: true,
-          selectedRowKeys,
+          selectedRowKeys: settings.row,
```

Comment:
question: should this be `settings.row || []`?

Reply:
`selectedRowKeys` can be undefined based on the typescript interface:
```
export interface TableRowSelection<T> {
  ...
  selectedRowKeys?: Key[];
  ...
}
```